package.json

```diff
@@ -35,8 +37,11 @@
     "@babel/plugin-proposal-class-properties": "^7.4.4",
     "@babel/polyfill": "^7.4.4",
     "@babel/preset-env": "^7.4.4",
+    "@typescript-eslint/eslint-plugin": "^1.12.0",
+    "@typescript-eslint/parser": "^1.12.0",
     "ansi-escapes": "^3.2.0",
     "asyncro": "^3.0.0",
+    "babel-eslint": "^10.0.2",
```

Comment:
does this necessary? 
I think use the `@typescript-eslint/parser` parser is enough.